recipe/recipe.yaml

```
@@ -0,0 +1,61 @@
+context:
+  name: "qdax"
+  version: 0.4.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/qdax-${{ version }}.tar.gz
+  sha256: 28bda31d7c65cdfe0c177b75176341922906efc8ec4f3a30a374ef996661c9cd
+  patches:
+    - fix-gym-dep.patch
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python ${{ python_min }}
```

```suggestion
    - python ${{ python_min }}.*
```

This way it should work - I tried to push to your branch but didn't work.

---

Thank you @wolfv!